dashboard/.storybook/preview.js

```diff
@@ -0,0 +1,22 @@
+import {ChakraProvider} from '@chakra-ui/react'
+import {StoryContext} from '@storybook/react'
+
+const withChakra = (StoryFn: Function, context: StoryContext) => {
+  return (
+    <ChakraProvider>
+      <StoryFn />
+    </ChakraProvider>
+  )
+}
```

Comment:
This decorator wraps all stories with `ChakraProvider` so styles are applied to components.